README.md

```diff
@@ -41,10 +41,7 @@ def project do
 
       # ...
     ],
-    application: [
```

Comment:
Why? It's one of the easier things to forget in Elixir, especially for beginners.